client/components/category-pill-navigation/index.tsx

```diff
@@ -43,8 +43,12 @@ export const CategoryPillNavigation = ( {
 		}
 
 		const { scrollLeft, scrollWidth, clientWidth } = listInnerRef.current;
-		setShowLeftArrow( scrollLeft > 0 );
-		setShowRightArrow( Math.ceil( scrollLeft ) < scrollWidth - clientWidth );
+
+		const roundedScrollLeft = Math.floor( scrollLeft );
+		const scrollLeftAbs = Math.abs( roundedScrollLeft ); // adjust RTL negative values
+
+		setShowLeftArrow( scrollLeftAbs > 0 );
+		setShowRightArrow( scrollLeftAbs + 1 < scrollWidth - clientWidth ); // +1 to account for rounding errors
```

Comment:
I didn't notice any perceivable difference with `+1`.  Can you expatiate on the rounding errors?

Reply:
Chrome (MacOS), at least for me - at the end of the scroll it shows 0.5px less value than it should be. It's connected with integer value for width, scrollLeft etc, Chrome rounds it.